### PR TITLE
Add component for slack token input

### PR DESCRIPTION
**Pull request by clairewild**
Screen Shot 2019-08-15 at 2 23 17 PM

### CODE REVIEW

File: emoji-ui/src/components/SlackTokenInput.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  handleSubmit(e) {
+    e.preventDefault();
+    // saving doesn't actually do anything yet!
+  }
+
+  render() {
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <label className="token-input-section">
+          Source Slack Api Key
```
Comment (gmkohler) [inline emoji-ui/src/components/SlackTokenInput.js]: 💡 Any chance we can add help text to tell them how to get this (the instructions are in the README)

**Comment by gmkohler** [comment]
@clairewild is this now being sent down to the backend?  I can adjust the code to accept these parameters.

File: emoji-ui/src/components/SlackTokenInput.js (hunk 1 above)
Comment (clairewild) [reply emoji-ui/src/components/SlackTokenInput.js]: Oh yeah, thought of that and then forgot :-)

File: emoji-ui/src/components/SlackTokenInput.js [hunk 2]
```diff
@@ -49,6 +49,12 @@ class SlackTokenInput extends Component {
           />
         </label>
 
+        <small className="token-input-help-text">
+          API keys can be found under window.TS.boot_data.api_token
+          while inspecting the "customize slack" webpage of a slack
+          instance. They should start with xoxs-*.
+        </small>
+
         <input className="token-input-submit" type="submit" value="Save" />
```
Comment (clanam) [inline emoji-ui/src/components/SlackTokenInput.js]: we'll probably want to save these in redux and pass them with the get/post calls (this is a note for later)

File: emoji-ui/src/components/emojiItem.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import './emojiItem.css';
+import './EmojiItem.css';
```
Comment (clanam) [inline emoji-ui/src/components/emojiItem.js]: is this change correct? git isn't recognizing the file name as having been changed.

File: emoji-ui/src/components/SlackTokenInput.js (hunk 2 above)
Comment (clairewild) [reply emoji-ui/src/components/SlackTokenInput.js]: yep right now the save button doesn't do anything

**Pull request merged**